tests/lib/illegal-eslint-disable-line.js

```diff
@@ -60,7 +60,7 @@ describe("multi-line eslint-disable-line comments", () => {
         const selfPath = path.resolve(__dirname, "../../")
         const pluginPath = path.resolve(
             __dirname,
-            "../../node_modules/eslint-plugin-eslint-comments"
+            "../../node_modules/@eslint-community/eslint-plugin-eslint-comments"
```

Comment:
I think we probably need to create a `"../../node_modules/@eslint-community"` directory after this.